ecs/build.zig

```diff
@@ -1,14 +1,12 @@
 const std = @import("std");
 
 pub fn build(b: *std.build.Builder) void {
-    const mode = b.standardReleaseOptions();
-
-    const main_tests = b.addTest("src/main.zig");
-    main_tests.addPackage(pkg);
-    main_tests.setBuildMode(mode);
-
     const test_step = b.step("test", "Run library tests");
-    test_step.dependOn(&main_tests.step);
+    test_step.dependOn(&testStep(b).step);
```

Comment:
I don't see `setBuildMode` being called anymore here, so this means we cannot test in release/debug modes easily now, right? I think we should keep that if possible